src/Mutation/MutationGenerator.php

```diff
@@ -99,15 +96,18 @@ public function __construct(
      */
     public function generate(bool $onlyCovered, array $nodeIgnorers): iterable
     {
-        $numberOfFiles = IterableCounter::bufferAndCountIfNeeded($this->sourceFiles, $this->runConcurrently);
+        $coveredFiles = $this->coveredFileDataProvider->provideFiles();
```

Comment:
It's a little bit confusing that variable and class named `covered...`, but actually we can provide uncovered files here as well:

```php
# CoveredFileDataFactory

// ...

if ($this->onlyCovered === true) {
    // The case where only covered files are considered
    return $readyCoverageFeed;
}

return $this->appendUncoveredFiles($readyCoverageFeed);
              ^^^
```

I would suggest to rename it to something that covers both covered an uncovered cases.

Reply:
Yes, true, I probably left these named this way after `CoveredFileData` objects, which, as I see it, should have a cousin `UnoveredFileData` following the same interface, but with slightly simplified internals. 

How does that feel if we'd name them `SourceFileDataSomething`?

Reply:
In this case here we can name `$coveredFiles` to `$files` though

Reply:
Alright, done